hooks/boot/04-services.rb

```diff
@@ -0,0 +1,36 @@
+require 'open3'
+
+module ServicesHookContextExtension
+  ALL_POSSIBLE_SERVICES = [
+    'httpd.service', 'apache.service', # RH vs Debian
+    'postgresql*',
+    'foreman*',
+    'dynflow*',
+    '*mongo*',
+    '*redis*',
+    'pulp*',
```

Comment:
If we are considering all services that are stopped as of today:

 * tomcat
 * puppet
 * puppetserver
 * smart_proxy_dynflow_core

Things going away soon:

 * qpidd
 * qdrouterd
 * squid

Reply:
Can you add tomcat to the list?